frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -332,17 +358,26 @@ export const PersistentVolumeClaimsPage = (props) => {
       : {};
   dispatch(setPVCMetrics(pvcMetrics));
 
-  const createProps = {
-    to: `/k8s/ns/${props.namespace || 'default'}/persistentvolumeclaims/~new/form`,
+  const isKubevirt = useFlag(FLAG_KUBEVIRT);
+  const to = `/k8s/ns/${props.namespace || 'default'}/persistentvolumeclaims/~new/form`;
+  const upload = `/k8s/ns/${props.namespace || 'default'}/persistentvolumeclaims/~new/upload-form`;
+  const createProps = { to };
+  const withCDICreateProps = {
+    items: {
+      blank: 'New with Form',
+      upload: 'New with Data Upload',
+    },
+    createLink: (wizardName) => (wizardName === 'blank' ? to : upload),
   };
+
   return (
     <ListPage
```

Comment:
we should ask users if they want to delete the DV as well  in PVC delete dialog as it is hard for users to delete the PVC if they don't know it is controlled by DV.

Reply:
Note, if the DV is not deleted, CDI will create a new PVC and associated pod (import/upload/clone)